zeroconf/asyncio.py

```diff
@@ -55,6 +55,35 @@ def _get_best_available_queue() -> queue.Queue:
     return queue.Queue()
 
 
+# Switch to asyncio.wait_for once https://bugs.python.org/issue39032 is fixed
+async def wait_condition_or_timeout(condition: asyncio.Condition, timeout: float) -> None:
```

Comment:
This can go away once cpython fixes the bug since we can replace it with `asyncio.wait_for` and suppress any timeout exceptions that get raised